doc/en/reference/plugin_list.rst

```diff
@@ -23,7 +23,7 @@ This list contains 969 plugins.
    :pypi:`pytest-addons-test`                       用于测试pytest的插件                                                                                                                                                      Aug 02, 2021    N/A                    pytest (>=6.2.4,<7.0.0)
```

Comment:
this file is generated and should not be touched

Reply:
To elaborate: Those descriptions are coming from the `setup.py` (or whatever `setup.py` replacement the project in question uses), so changing it here means it will just get overwritten on the next update of the list. The right place to fix those would be the projects themselves.